pkg/client/client_test.go

```diff
@@ -523,6 +523,9 @@ func TestRead(t *testing.T) {
 		Kind: "temperature",
 		Data: []scheme.ReadData{
 			scheme.ReadData{
+				// FIXME: If we change the type of value to anything other than
```

Comment:
should open an issue for this, providing the failure message you get. definitely seems like weird behavior